### DISCUSSION TITLE
I can't connect APP with BMW G20

### DISCUSSION THREAD

**Discussion by rsuarezt** [BimmerGestalt/AAIdrive#603] [General]
Good morning,
![chrome_GNArbZwiHf](https://user-images.githubusercontent.com/111573801/185612545-e253cea2-1345-4656-b1d3-a8f9e3b55b07.png)
![chrome_eSvoCzz3fY](https://user-images.githubusercontent.com/111573801/185612845-031dbd97-32dc-4f89-8898-55e3f7abe9fc.png)

I have a BMW G20 (March 2019 - ID6) and I can't connect the app with the car.

It tells me "Enable the option of applications in the car for this mobile"

I have a conectDrive subscription and I have the MyBMW app activated and connected to the car.

I have re-synchronized the phone via BT but there is no "APPS" option.

Do you know why this could be happening?

All the best

**Comment by hufman** [comment]
I'm not sure, having the ConnectedDrive subscription should be enough. Did you enable the subscription only recently? It sometimes takes a few days for the car itself to update the subscription status and show the Apps box. Does MyBMW show the Apps checkbox in the app too?

![photo1621640827](https://user-images.githubusercontent.com/1592375/185658485-0e055554-1bac-48c4-bccf-5438f7f0764c.jpeg)

**Reply by rsuarezt** [reply]
Hello,
Thanks for your answer.
I have already solved.
The problem was that with the new version of MYBMW the APPs box in ConnectDrive does not appear.
I have downloaded an APK of the previous version and it already works.
In fact, the option of locating the vehicle and others has also appeared.

The AAIdrvie is amazing. It works great.

I repeat, thank you for your response.

**Reply by hufman** [reply]
That's very weird! Do you know what version of MyBMW you had before and are using now?

**Comment by rsuarezt** [comment]
I was using the latest version before.
Right now I'm on 1.7.4 and it works perfectly.
